content/rest-api/filter.md

```diff
@@ -160,6 +160,139 @@ To get the products that were updated during the last 4 days, you can use the fo
+
+```
+/api/rest/v1/product-models?search={"enabled":[{"operator":"IN","value":["winter_collection"}],"completeness":[{"operator":"ALL COMPLETE","locale":["en_US"]}]}
+```
+
+You can even combine several filters on the same product model properties. The example below will get you the product models created both the 4th and the 5th of July 2016.
+
+```
+/api/rest/v1/product-models?search={"created":[{"operator":"=","value":"2016-07-04"},{"operator":"=","value":"2016-07-05"}]}
+```
+
+### On categories
+
+To filter product models on their categories, use the property `categories`.
+Here are the allowed operators you can use to filter on the category code as well as the corresponding type of value expected in the `search` query parameter.
+
+| Operator | Allowed value type | Filter description |
+| ----------------- | -------------- | ------------------ |
+| `IN` | an array of existing category codes | Only returns the product models that are in the given categories |
+| `NOT IN` | an array of existing category codes | Only returns the product models that are not in the given categories |
+| `IN OR UNCLASSIFIED` |  an array of existing category codes | Only returns the product models that are in the given categories or that are not classified in any categories |
+| `IN CHILDREN` | an array of existing category codes | Only returns the product models that are in the children of the given categories |
+| `NOT IN CHILDREN` | an array of existing category codes | Only returns the product models that are not in the children of the given categories |
+| `UNCLASSIFIED` | no value | Only returns the product models that are not classified into any category |
+
+#### Example
+To get the product models of the `winter_collection` category, you can use the following URL.
+
+```
+/api/rest/v1/product-models?search={"categories":[{"operator":"IN","value":["winter_collection"]}]}
+```
+
+### On completeness
+
+To filter product models on their completeness, use the `completeness` product property. You will also need to provide a `scope` value to specify on which channel you want to filter with the completeness.
+Here are the allowed operators you can use to filter by completeness as well as the corresponding type of value expected in the `search` query parameter.
+
+| Operator | Allowed value type | Filter description |
+| ----------------- | -------------- | ------------------ |
+| `AT LEAST COMPLETE` | no value | Only returns product models that have at least one variant product fully complete on the given channel for the given set of locales |
+| `ALL COMPLETE` | no value |  Only returns product models that have all his variant products fully complete on the given channel for the given set of locales |
+
+#### Examples
+To get the product models that are 100% complete for the `ecommerce` channel, you can use the following URL.
+
+```
+/api/rest/v1/product-models?search={"completeness":[{"operator":"COMPLETE","scope":"ecommerce"}]}
```

Comment:
This example does not work.
It should be:
`/api/rest/v1/product-models?search={"completeness":[{"operator":"ALL COMPLETE","locale":"fr_FR","scope":"ecommerce"}]}

Reply:
fix